content/src/roadmaps/license-tasks/authorization-architect-firm.md

```diff
@@ -1,5 +1,5 @@
 ---
```

Comment:
So in the AC it says that these should only be added to road map with an entered address and business name. 

I just checked in a completely up and running state (via the cards). But I didn't need to enter an address or business name is this intended?

Reply:
Yes that is fine